skimage/measure/tests/test_find_contours.py

```diff
@@ -9,6 +9,9 @@
 a[1:-1, 1] = 0
 a[1, 1:-1] = 0
 
+b = np.copy(a)
+b[7:, :] = np.nan
```

Comment:
Maybe move this into the `test_nodata` function if it is only used there?

Reply:
I was following the same pattern as the existing tests in `test_find_contour.py` here: all the test data is constructed at the top of the file. (For example, `a`, despite only being used in `test_binary`.)